OmniCore.Radio/OmniCore.Radio.RileyLink/RileyLinkMessageExchangeProvider.cs

```diff
@@ -17,8 +17,23 @@ public class RileyLinkMessageExchangeProvider : IMessageExchangeProvider
         private static RileyLinkMessageExchange RileyLinkMessageExchange;
 
         [method: SuppressMessage("", "CS1998", Justification = "Not applicable")]
-        public async Task<IMessageExchange> GetMessageExchange(IMessageExchangeParameters messageExchangeParameters, IPod pod)
+        public async Task<IMessageExchange> GetMessageExchange(IMessageExchangeParameters messageExchangeParameters, IPod pod, IMessageExchangeProgress messageProgress)
```

Comment:
design conflict again, messageexchange contains the progress not the other way around. I like the idea of provider verifying connectivity, but in that case more changes are needed - the way it is, it violates the separation of concerns principle

Reply:
Agree.
Was the easiest way to implement the change for rebuilding the RileyLinkInstance in GetMessageExchange(). Did not want to rewrite/redesign code.
Just "proof of concept" on "rebuilding RileyLinkInstance in GetMessageExchange()."